examples/How_to_count_tokens_with_tiktoken.ipynb

```diff
@@ -38,15 +38,15 @@
     "\n",
     "For `o200k_base`, `cl100k_base` and `p50k_base` encodings:\n",
     "- Python: [tiktoken](https://github.com/openai/tiktoken/blob/main/README.md)\n",
-    "- .NET / C#: [SharpToken](https://github.com/dmitry-brazhenko/SharpToken), [TiktokenSharp](https://github.com/aiqinxuancai/TiktokenSharp)\n",
+    "- .NET / C#: [Microsoft.ML.Tokenizers](https://www.nuget.org/packages/Microsoft.ML.Tokenizers/), [TiktokenSharp](https://github.com/aiqinxuancai/TiktokenSharp)\n",
```

Comment:
Should we list the data package links? like 

https://www.nuget.org/packages/Microsoft.ML.Tokenizers.Data.O200kBase
https://www.nuget.org/packages/Microsoft.ML.Tokenizers.Data.Cl100kBase
https://www.nuget.org/packages/Microsoft.ML.Tokenizers.Data.P50kBase

and below for gpt2 use:

https://www.nuget.org/packages/Microsoft.ML.Tokenizers.Data.R50kBase
https://www.nuget.org/packages/Microsoft.ML.Tokenizers.Data.Gpt2

Reply:
To tell, when referencing the data packages, no need to explicitly referencing `Microsoft.ML.Tokenizers` .

Reply:
Good point. I'll make that change.